### PR TITLE
Octopus ID

### PR DESCRIPTION
**Pull request by slewis74**
Initial page content for Octopus ID authentication provider

See also OctopusDeploy/Octopurls/pull/39

### CODE REVIEW

File: docs/administration/authentication/octopusid-authentication.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Octopus ID Authentication
+description: Octopus Deploy can use Octopus accounts to identify users.
+position: 7
+version: 2019.?.????
```
Comment (Waldo000000) [inline docs/administration/authentication/octopusid-authentication.md]: Should this be a real version, or are the `?`s intentional?

File: docs/administration/authentication/octopusid-authentication.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+:::hint
+Octopus ID authentication is available only in **Octopus Cloud**
+:::
+
+Octopus ID authentication allows you to log in to your Octopus hosted instance using your Octopus account.
```
Comment (Waldo000000) [inline docs/administration/authentication/octopusid-authentication.md]:
```suggestion
Octopus ID authentication allows you to log in to your Octopus Cloud instance using the same account that you use to sign in at octopus.com.
```

File: docs/administration/authentication/octopusid-authentication.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+Support for Octopus ID will be progressively rolled out to Octopus Cloud, and configuration is handled by the hosted Octopus infrastructure so no user configuration is required.
```
Comment (Waldo000000) [inline docs/administration/authentication/octopusid-authentication.md]:
```suggestion
Support for Octopus ID authentication will be progressively rolled out to Octopus Cloud instances, and configured automatically.

For more information about using your Octopus Cloud instance, see our [Octopus Cloud](https://octopus.com/docs/octopus-cloud) documentation.
```

File: docs/administration/authentication/octopusid-authentication.md (hunk 1 above)
Comment (wordlee) [reply docs/administration/authentication/octopusid-authentication.md]: We don't really use version any more (the functionality still exists, but that's more for partials than anything else). I think we can remove the version from the YAML, and when/if there's a need to specify the version we can do so in the body of the text.

File: docs/administration/authentication/octopusid-authentication.md (hunk 3 above)
Comment (wordlee) [inline docs/administration/authentication/octopusid-authentication.md]: minor grammar tweak:
```suggestion
Support for Octopus ID will be progressively rolled out to Octopus Cloud. Configuration for this feature is handled by the hosted Octopus infrastructure so no user configuration is required.
```

File: docs/administration/authentication/octopusid-authentication.md (hunk 1 above)
Comment (slewis74) [reply docs/administration/authentication/octopusid-authentication.md]: Ah, thanks, I'll remove it then

**Pull request merged**